### DISCUSSION TITLE
Dark mode in web-gui

### DISCUSSION THREAD

**Discussion by Ontrackx** [UtilitechAS/amsreader-firmware#386] [Ideas]
Can it be possible  choose dark mode in web -gui in the future? 

Im blind :)

**Comment by gskjold** [comment]
This have been requested a few times. I am currently working on a new version, so I will look into the possibility of having this. I need to consider how much extra memory this option will consume before I make a final decision.

**Comment by ElMyggo** [comment]
I also have a wish for a dark theme :-) maybe 2 different binary
or i have to test compile my own binary...

**Reply by gskjold** [reply]
Noted! Best if this is compiled in with the default firmware. This will not be included in the upcoming v2.2, but will be included at a later point. Feel free to build your own though :)

**Reply by ElMyggo** [reply]
ok! i have to try building my own then... puuuh , even if my knowledge is low on this, but learning by doing :-)

**Reply by ElMyggo** [reply]
Now i have compiled my first firmware file so can get a hint on which file/files to modify to change colors
in web folder boot.css ?

**Reply by gskjold** [reply]
Judging from your comment, you are compiling the master branch which is using bootstrap from CDN. This means that there are no CSS files in the project which is used for the main dashboard. You would need to change classes on the elements based on best practice for dark mode with bootstrap to change the colors.

In the upcoming v2.2, the original dashboard have been recreated using Tailwind CSS, so the implementation would be different there.

**Reply by ElMyggo** [reply]
I wait for the 2.2 then, i testing a bit with a testrigg and i came up with this on the first attemp

![first](https://user-images.githubusercontent.com/111691693/214109114-58dc6b35-ec32-4fe3-9311-6892e0fd8d9e.jpg)
